apps/accounts/api/v1/serializers.py

```
@@ -0,0 +1,106 @@
+    class Meta:
+        model = User
+        fields = [
+            "id",
+            "display_name",
+            "username",
+            "email",
+            "password",
+            "img_url",
+            "about",
+            "pscore",
+        ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)